app/services/idv/session.rb

```diff
@@ -30,6 +30,7 @@ class Session
       vendor_phone_confirmation
       verify_info_step_document_capture_session_uuid
       welcome_visited
+      skip_doc_auth
```

Comment:
Please insert in alphabetical order, after `resolution_successful`.

Reply:
Done. Resolved with commit 158cb19f98667c682ae19e98f2e427037e60ce82